cloudformation.yaml

```diff
@@ -32,6 +32,12 @@ Resources:
                 - logs:PutLogEvents
                 - lambda:InvokeFunction
               Resource: "*"
+        - PolicyName: ReadPrivateCredentials
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action: s3:GetObject
+              Resource: !Sub arn:aws:s3:::fulfilment-lookup-private/${Stage}/*
```

Comment:
I'm not sure if we should share membership-private and use a separate dir, or a new bucket (I had this issue with the supporter-metrics-job lambda which I should probably clean up from AWS)

Reply:
Yes, I opted for following current convention after some deliberation... however, current convention does produce a lot of tiny S3 buckets - we have a similar issue with everything having it's own dist bucket! Perhaps we need a membership AWS account spring clean...

Reply:
Ah, I see now that there isn't even a consistent convention

Reply:
yep agreed about the dist buckets!  maybe we just need a general labmdas-private and lambdas-dist